slurm_ops_manager/slurm_snap_manager.py

```diff
@@ -123,7 +123,9 @@ def upgrade(self):
         # instead).
         self.setup_system()
 
-    def _provision_snap_systemd_service_override_file(self):
+    def write_systemd_file(self):
```

Comment:
move this function needs to `slurm_ops_base.py`, rename it to be `write_slurm_process_systemd_override_conf()`